pkg/booleanpolicy/field_metadata.go

```diff
@@ -769,9 +769,9 @@ func initializeFieldMetadata() FieldMetadata {
 
 	f.registerFieldMetadata(
 		fieldnames.KubeResourceName,
-		querybuilders.ForFieldLabel(augmentedobjs.KubernetesResourceNameCustomTag), nil,
+		querybuilders.ForFieldLabelRegex(augmentedobjs.KubernetesResourceNameCustomTag), nil,
```

Comment:
@theencee This is curious, I wonder if this is forwards compatible. 🤔 it will work with regex if we do r/ if I keep the original, and this will automatically make it a regex

Reply:
Tempted to leave it as is and require the use to use r/ for fear of breaking existing policies

Reply:
WOuldn't this fail on already existing policies without proper migration? And yes, roll backs can cause a problem.

Reply:
Rolled it back